dsprotfrontend/dsprot-frontend/src/components/Transaction.vue

```diff
@@ -3,7 +3,7 @@
     Property 'transaction' is used to determine this. -->
     <div :class="[transaction.isValid ? 'valid' : 'invalid', 'transaction']">
         <h3>ID: {{transaction.tid}}</h3>
-        <p>Points spent: {{transaction.spendAmount}}</p>
+        <p>Points spent: {{transaction.userChoice}}</p>
     </div>
 </template>
 
```

Comment:
I guess you forgot to delete the old frontend folder? :thinking:

Reply:
Yes, old folder will be gone in the next commit